core/qbft/qbft.go

```diff
@@ -72,13 +75,22 @@ const (
 
 // Msg defines the inter process messages.
```

Comment:
suggest: replace word `process` with `nodes` as it is confusing since we deal with Charon **nodes** not processes

Reply:
the algorithm paper talks about processes (so sticking with its terminology)

Reply:
yeah sure, paper talks about processes but we are dealing with nodes not processes so we can/should modify terminologies as per our usecases